From 15d41b84bca17d554053c4726ff57b1a16ead28b Mon Sep 17 00:00:00 2001
From: Phil Sutter <psutter@redhat.com>
Date: Thu, 18 Feb 2016 15:29:58 +0100
Subject: [PATCH] man: fix whatis for fq

Bugzilla: https://bugzilla.redhat.com/show_bug.cgi?id=1261520
Upstream Status: iproute2.git commit e947d8947d341

commit e947d8947d34114e2ea7c5508dfbb9b10b4611c2
Author: Stephen Hemminger <stephen@networkplumber.org>
Date:   Wed Jan 6 10:29:06 2016 -0800

    man: fix whatis for fq

    The FQ man page was not following whatis formatting rules.
---
 man/man8/tc-fq.8 | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/man/man8/tc-fq.8 b/man/man8/tc-fq.8
index 993beb6..f058a05 100644
--- a/man/man8/tc-fq.8
+++ b/man/man8/tc-fq.8
@@ -1,6 +1,6 @@
 .TH FQ 8 "10 Sept 2015" "iproute2" "Linux"
 .SH NAME
-Fair Queuing (FQ) \- Traffic Pacing
+FQ \- Fair Queue traffic policing
 .SH SYNOPSIS
 .B tc qdisc ... fq
 [
-- 
1.8.3.1

